src/styles/global.css

```
@@ -23,6 +23,9 @@ h1 {
 /* nav styles */
 
 .hamburger {
+  background: transparent;
+  display: block;
+  border: none;
   padding-right: 20px;
   cursor: pointer;

```

```suggestion
  background-color: #0d0950;
  border: none;
  color: #fff;
  font-size: 1.2rem;
  font-weight: bold;
  padding: 5px 10px;
```

We slightly tweak the appearance of the new menu button.